var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -6,6 +6,7 @@
 from spack import *
 import glob
 import os
+import platform
```

Comment:
We try to not rely on `platform` for architecture specific decisions, but rather on `spec.target` which is the target of compilation.

Reply:
Done